### ISSUE TITLE
Can't import SentryInterceptor

### ISSUE THREAD

**Issue by maxpeterson** [ntegral/nestjs-sentry#20]
It looks like there may be an issue with the version 1.0.4 release / build.

Based on [lib/services/index.ts#L2](https://github.com/ntegral/nestjs-sentry/blob/59fd98e79fa0bdfbec271295261cc649ce8367e7/lib/services/index.ts#L2) I would expect to be able to import `SentryInterceptor`:

```
import { SentryInterceptor } from '@ntegral/nestjs-sentry';
```

However this fails. Inspecting the `dist` I noticed that `SentryInterceptor` is not exported from [dist/services/index.js](https://github.com/ntegral/nestjs-sentry/blob/59fd98e79fa0bdfbec271295261cc649ce8367e7/dist/services/index.js)

I have re-built the `dist` locally and the  `SentryInterceptor` can then be imported ands works as expected.

I am not sure if it is a release issue, but would it be possible to create a new version with the `SentryInterceptor` exported.

**Comment by maxpeterson** [comment]
If `SentryInterceptor` is not intended to be exported then please just close this issue!

**Comment by kanidjar** [comment]
Same here, was quite surprised not to be able to import the SentryInterceptor

**Comment by davids-sparkshare** [comment]
This seems to have been fixed in #23

**Comment by ntegral** [comment]
This has been updated
